src/db_ref.ts

```diff
@@ -111,17 +112,26 @@ export class DBRef extends BSONValue {
     return new DBRef(doc.$ref, doc.$id, doc.$db, copy);
   }
 
-  /** @internal */
-  [Symbol.for('nodejs.util.inspect.custom')](): string {
-    return this.inspect();
-  }
+  inspect(depth?: number, options?: unknown, inspect?: InspectParameterFn): string {
+    const addQuotes = inspect ? false : true;
```

Comment:
```suggestion
    const addQuotes = !inspect;
```

Reply:
this applies to all occurrences of this logic in the PR

Reply:
I'll be sure to change this for my next commit!